SSM/GameManagers/GameManager.java

```diff
@@ -611,7 +645,7 @@ public void PlayerJoin(PlayerJoinEvent e) {
             KitManager.equipPlayer(e.getPlayer(), new KitTemporarySpectator());
             return;
         }
-        e.getPlayer().teleport(lobby_world.getSpawnLocation());
+        SSM.teleportAllPlayers();
```

Comment:
Why are you teleporting all players when someone joins?

Reply:
lol i noticed that when we played yesterday and i already fixed that just remove it